src/components/FeatureViews/Registration/RegistrationCart.js

```diff
@@ -38,7 +38,7 @@ const RegistrationCart = () => {
 
   const courseIDs = useMemo(
 	  () => {
-		  if (registered_courses.length > 0) {
+		  if (Object.values(registered_courses).length > 0) {
```

Comment:
Unless you specifically want `courseIDs` to be undefined if there are no registered courses, remove this `if` statement and `courseIDs` will simply be an empty array if there are no registered courses -  `Object.values` returns an empty array, and the subsequent operations do not change the empty array.